pageserver/src/tenant/mgr.rs

```diff
@@ -1108,7 +1108,7 @@ impl TenantManager {
                 // the caller thinks they're creating but the tenant already existed.  We must switch to
                 // Normal mode so that when starting this Tenant we properly probe remote storage for timelines,
```

Comment:
nit: this comment mentions the old 'Normal' value

Reply:
Good spotting, Ill fix it.

Reply:
[fb23c75](https://github.com/neondatabase/neon/pull/6907/commits/fb23c75bd1da882a35812beca2df6d97d4f05efc)